blockscout-ens/graph-node/subgraphs/zns-ink-subgraph/tests/zns-registry.test.ts

```
@@ -0,0 +1,62 @@
+import {
+  assert,
+  describe,
+  test,
+  clearStore,
+  beforeAll,
+  afterAll
+} from "matchstick-as/assembly/index"
+import { Address, BigInt, Bytes } from "@graphprotocol/graph-ts"
```

Biome (1.9.4)

[error] Do not shadow the global "BigInt" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)